pom.xml

```diff
@@ -104,6 +104,11 @@
       <version>0.9.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
No new non-test dependencies should be added, ideally no new test dependencies either.
But this does not even seem to be needed?